public/502.html

```diff
@@ -14,120 +14,125 @@
+              <li id="medium_8"><a href="/media/Other%20Media/fandoms">Other Media</a></li>
+              <li id="medium_30198"><a href="/media/Theater/fandoms">Theater</a></li>
+              <li id="medium_1"><a href="/media/TV%20Shows/fandoms">TV Shows</a></li>
+              <li id="medium_476"><a href="/media/Video%20Games/fandoms">Video Games</a></li>
+              <li id="medium_9971"><a href="/media/Uncategorized%20Fandoms/fandoms">Uncategorized Fandoms</a></li>
+            </ul>
+          </li>
+          <li class="dropdown">
+            <a href="/menu/browse">Browse</a>
+            <ul class="menu" role="menu">
+              <li><a href="/works">Works</a></li>
+              <li><a href="/bookmarks">Bookmarks</a></li>
+              <li><a href="/tags">Tags</a></li>
+              <li><a href="/collections">Collections</a></li>
+            </ul>
+          </li>
+          <li class="dropdown">
+            <a href="/menu/search">Search</a>
+            <ul class="menu" role="menu">
+              <li><a href="/works/search">Works</a></li>
+              <li><a href="/people/search">People</a></li>
+              <li><a href="/bookmarks/search">Bookmarks</a></li>
+              <li><a href="/tags/search">Tags</a></li>
+            </ul>
+          </li>
+          <li class="dropdown">
+            <a href="/menu/about">About</a>
+            <ul class="menu" role="menu">
+              <li><a href="/about">About Us</a></li>
+              <li><a href="/admin_posts">News</a></li>
+              <li><a href="/archive_faqs">FAQ</a></li>
+              <li><a href="/donate">Donate or Volunteer</a></li>
+            </ul>
+          </li>
+          <li class="search"><form accept-charset="UTF-8" action="/works/search" class="search" id="search" method="get"><div style="margin:0;padding:0;display:inline"><input name="utf8" type="hidden" value="&#x2713;" /></div>
+            <fieldset>
+              <legend>Search Works</legend>
+                <p>
+                  <label class="landmark" for="site_search">Work Search:</label>
+                  <input aria-describedby="site_search_tooltip" class="text" id="site_search" name="work_search[query]" type="text">
+                  <span class="tip" role="tooltip" id="site_search_tooltip">tip: lex m/m (mature OR explicit)</span>
+                  <span class="submit actions"><input class="button" type="submit" value="Search" /></span>
+                </p>
+              </fieldset>
+            </form>
+          </li>
+        </ul>
+        <div class="clear"></div>
+      </div>
+      <!-- END header -->
+      <div id="inner" class="wrapper">
+        <div id="main" class="system errors error-502 region">
+          <h2 class="heading">Error 502</h2>
+          <h3 class="heading">The page was responding too slowly.</h3>
+          <!--plain english suggestion: There are so many people using the archive right now, we can't show your page. -->
```

Comment:
Do we want to implement the plain English suggestion or remove the comment if we're not?